### ISSUE TITLE
option to disable the <div class="optiscroll-h"> markup

### ISSUE THREAD

**Issue by neuropass** [albertogasparin/Optiscroll#41]
is there a way to tell optiscroll not to inject the horizontal markup? 

```<div class="optiscroll-h"><b class="optiscroll-htrack"></b></div>```

or vice-versa in the only horizontal is used.

basically inject only what's in use.

**Comment by albertogasparin** [comment]
No, given they are just two elements, I considered add/removing them on scroll change more expensive than leaving them always in the dom. 
If you are having issue with styling, you can force `display: none` on `optiscroll-h` and then override it when `has-vtrack` `has-htrack` classes are added to `optiscroll` main element (instead of the default `visibility: hidden` behaviour).
